src/clients/python/__init__.py

```diff
@@ -1085,6 +1091,16 @@ def _prepare_request(self, inputs, outputs,
                                         _crequest_infer_ctx_input_set_raw(
                                             input, input_value.ctypes.data_as(c_void_p),
                                             c_uint64(input_value.size * input_value.itemsize))))
+                    elif isinstance(input_values[1], (list, tuple)) and type(input_values[0]) == c_void_p:
```

Comment:
What is this case? Add a comment. Use parens around sub-expressions in the if

Reply:
For variable size tensors, need the input shape as well as the shared memory handle